plugins/feeds/public/abusech_malwarebazaar.py

```diff
@@ -53,22 +76,36 @@ def analyze(self, block):
         context["imphash"] = block["imphash"]
         context["ssdeep"] = block["ssdeep"]
         context["date_added"] = datetime.utcnow()
+        tags = []
 
-        malware_file = File.get_or_create(value="FILE:{}".format(block["sha256_hash"]))
+        file_value = "FILE:{}".format(block["sha256_hash"])
+        
+        malware_file = observable.Observable.find(value=file_value)
+        if not malware_file:
+            malware_file = observable.Observable(value=file_value, type="file").save()
+        logging.debug(f"Malware file: {malware_file} source: {self.name} context: {context}")
+        malware_file.add_context(self.name, context)
```

Comment:
You need to exclude the `date_added` field from the context comparison, so that it doesn't get duplicated every time the feed is run.
```suggestion
        malware_file.add_context(self.name, context, skip_compare={'date_added'})
```